### PR TITLE
Fix Ranking Criteria

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!/en.md [hunk 1]
```diff
@@ -17,18 +19,17 @@ General
 -   **Combo colors must not blend in with the map's background/storyboard/video.** This is so the approach circles are always visible to the player.
 -   **There must be a new combo either on or after a group of spinners**, so that the objects on opposing sides of the spinner(s) don't form a single combo. (osu! checks this automatically unless you are editing .osu files by hand.)
 
-Skinning
---------
+### Skinning
 
 -   **Spinner circles must have a clearly visible center** to make sure players have something to effectively spin around while playing through the map.
 -   **Spinner circles must be clearly visible** (i.e. they should not be excessively transparent or near-invisible). This will ensure that players have a visible aid to help them spin consistently.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!/en.md]: `i.e.` to `e.g.`

File: wiki/Ranking_Criteria/osu!/en.md [hunk 2]
```diff
@@ -17,18 +19,17 @@ General
...
 
 Guidelines
-==========
+-----------
 
 Guidelines are important and should be followed in most maps. However, they are NOT rules, so they may be broken in **special cases**. If you want to break a guideline, ask yourself this: Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines? If you answer yes to both these questions, then it is probably okay.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!/en.md]: replace entire line with these fixes
```
Guidelines are important and should be followed in most maps. However, they are **not** rules, so they may be broken in special cases. If you want to break a guideline, ask yourself this, "Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines?" If you answer yes to both these questions, then it is probably okay.
```

File: wiki/Ranking_Criteria/osu!/en.md [hunk 3]
```diff
@@ -17,18 +19,17 @@ General
...
+-   **Beat placement should follow the time-distance equality rule** (using [distance snap](/wiki/Beatmap_Editor/Distance_Snap) while placing beats enforces this). This means that you shouldn't have some fast notes with a huge distance between them followed by closely-placed (but slow) notes. This gets really confusing if the player isn't the person who designed the beatmap! Having some logic to the beat placement should be one of your goals.
 -   **Avoid placing hitobjects under the HP/life bar** at the top of the screen, as this can greatly impair the visibility of notes and approach circles to the player. It doesn't hurt to use the default skin to check this!
 -   **Try to use the same tickrate on every difficulty** as it is a property of the music rather than the mapping. Using high tickrates to increase score/combo/difficulty is senseless.
 -   **Use new combos often and intelligently.** Using a new combo every few beats wouldn't make sense, but you also want to avoid long combos that continue for many measures.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!/en.md]: line 37:
- `[Post Spinner Note](/wiki/Mapping_Techniques/Spinners/#post-spinner-note)`
- `[Pre Spinner Note](/wiki/Mapping_Techniques/Spinners/#pre-spinner-note)`

add a line to the end of the file

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 4]
```diff
@@ -1,21 +1,22 @@
+osu!catch Ranking Criteria
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: change to `osu!catch` explanation is at osu!

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
+=============================
+
 The **Catch the Beat Ranking Criteria** is a set of rules and guidelines that apply to the creation of Catch the Beat-specific difficulties. In order to get a Catch the Beat-specific difficulty ranked, it is mandatory the creation obeys to the listed criteria. While **all rules must be followed in any circumstance**, guidelines may be violated under exceptional circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: use osu!catch (and fix others)

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
+-   ![Normal icon](/wiki/shared/Normal-f.png "Normal icon") **Salad**. Normal
+-   ![Hard icon](/wiki/shared/Hard-f.png "Hard icon") **Platter**. Hard
+-   ![Insane icon](/wiki/shared/Insane-f.png "Insane icon") **Rain**. Insane
+-   ![Expert icon](/wiki/shared/Expert-f.png "Expert icon") **Overdose**. Expert
 
-Gameplay
---------
+### Gameplay
 
 -   **Fruit.** A large object represented by a hitcircle, slider head, tail or repeat.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: For this and other terms below:
- `**Fruit:**`
- `**Drop:**`
- `**Droplet:**`
- `**Banana:**`
- `**Jump:**`
- `**Hyperjump:**`
- `**Edge Dash:**`
- `**Trigger Distance:**`

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 7]
```diff
@@ -51,49 +51,46 @@ General
...
 -   **Hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.
 
-Skinning
---------
+### Skinning
 
-### Rules
+#### Rules
 
 -   **Custom catchers must be included in v2 skin format.** This is to ensure correct display on all skins. The required filenames are "fruit-catcher-idle.png", "fruit-catcher-kiai.png" and "fruit-catcher-fail.png".
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: use code for skin elements

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 8]
```diff
@@ -51,49 +51,46 @@ General
...
 -   **Custom fruits must include all necessary elements and be colored in a scale of grey colors.** This is to ensure that your images are clearly defined and of acceptable quality. The needed elements can be found in the osu! wiki </wiki/Skinning_Catch_the_Beat> . Additionally, it is recommendable to use transparent elements for the overlays.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: Change this sentence `The needed elements can be found in the osu! wiki </wiki/Skinning_Catch_the_Beat> .` to `The needed elements can be found at [Skinning/osu!catch](/wiki/Skinning/osu!catch).`

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 9]
```diff
@@ -51,49 +51,46 @@ General
...
 -   **Skinned elements must be the same size as their default skin counterparts.** This is so they represent the hitbox properly and don't alter gameplay. The current dimensions used in the default skin are 128x128 pixels for the fruits, 82x103 for the drops and 306x320 for the catcher.
 
-### Guidelines
+#### Guidelines
 
 -   **Custom catchers should additionally include the element "lighting.png" to complete the skin set.** This element is however optional to add and has default dimensions of 184x184 pixels, though it may vary depending on the desired visibility of the element.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!catch/en.md]: code for skin element

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 10]
```diff
@@ -1,19 +1,21 @@
+osu!mania Ranking Criteria
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: change to `osu!mania` explanation is at osu!

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 11]
```diff
@@ -1,19 +1,21 @@
...
+=============================
+
 For the record, these include the recent changes made [from the discussion forum](http://osu.ppy.sh/forum/87). Any rule that is being discussed is listed here as it used to be and will be updated once the discussion reaches an agreement.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: use `https`

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 12]
```diff
@@ -1,19 +1,21 @@
...
 
 -   Non-expert = Easy, Normal. (Basic, Standard)
 -   Expert = Hard, Insane. (Hyper, Another)
 -   Column = 5K, 5 column. The center column is sometimes called Special or Step column.
 
 Rules
-=====
+-------
 
 All rules are exactly that: RULES. They are NOT guidelines and may NOT be broken under ANY circumstance.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: CAPS to lower; use bold in place of the CAPS

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 13]
```diff
@@ -24,19 +26,17 @@ Considering most keyboards use USB connections which do not allow 7 simultaneous
 -   **You can't have two notes in the same column that overlap each other, and there must always be a space between two notes in the same column.**
 -   **You must turn off the letterbox option.** Letterboxing is not supported in osu!mania mode and therefore should not be used.
 
-Difficulty Spread
------------------
+### Difficulty Spread
 
 -   **Each beatmap must have at least 2 difficulties and one of them must be Easy/Normal. Marathon is an exemption**
 -   **Different key amounts within a mapset must be treated as different gamemodes and require a respective independent difficulty spread.** Therefore each key amount must have an independent difficulty spread. For example if you want to add a 7K difficulty to your set, you must design a whole 7K spread. If you want to add an additional 4k difficulty to your set, you must also design a whole 4K spread.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: the second to last `k` isn't capitalized

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 14]
```diff
@@ -24,19 +26,17 @@ Considering most keyboards use USB connections which do not allow 7 simultaneous
...
-------
+### Timing
 
 -   **You can't use uninherited timing sections when there is no actual BPM changes.** In other words, you can't use uninherited timing sections slowly to change the scrolling speed. Use inherited timing sections and SV changes instead.
 
 Guidelines
-==========
+-----------
 
 Guidelines are important and should be followed in most maps. However, they are NOT rules, so they may be broken in special cases. If you want to break a guideline, ask yourself this: Does what I’m about to do make sense? Is it more fun to play like this compared to sticking to the guidelines? If you answer yes to both these questions, then it is probably okay.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: replace entire line with these fixes
```
Guidelines are important and should be followed in most maps. However, they are **not** rules, so they may be broken in special cases. If you want to break a guideline, ask yourself this, "Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines?" If you answer yes to both these questions, then it is probably okay.
```

File: wiki/Ranking_Criteria/osu!taiko/en.md [hunk 15]
```diff
@@ -22,42 +23,24 @@ Gameplay
...
 
 All rules are exactly that: RULES. They are NOT guidelines and may NOT be broken under ANY circumstance.
 
 -   **Finish notes must not be in the middle or at the beginning of a 1/4 or above snapped pattern.** This is to ensure playability of all patterns, as finisher notes in the middle or at the beginning of a 1/4 or above snapped pattern have been deemed as very issuesome to catch.
 
 Guidelines
-==========
+--------------
 
 Guidelines are important and should be followed in most maps. However, they are NOT rules, so they may be broken in **special cases**. If you want to break a guideline, ask yourself this: Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines? If you answer yes to both these questions, then it is probably okay. When your guideline-breaking map gets modded, try to be as reasonable as possible: Do your best to explain your point of view and be open to suggestions.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/en.md]: replace entire line with these fixes
```
Guidelines are important and should be followed in most maps. However, they are **not** rules, so they may be broken in special cases. If you want to break a guideline, ask yourself this, "Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines?" If you answer yes to both these questions, then it is probably okay.
```

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 16]
```diff
@@ -74,11 +74,11 @@ Note: 1K, 2K, 3K, 9K <span style="text-decoration:line-through;">and 10K</span>
...
+| 10K                                                    | MuziBox, Pump It Up (Double)                                                    |
 | 12K                                                    | Keyboardmania (Light)                                                           |
 | 14K                                                    |                                                                                 |
 | 16K                                                    | beatmaniaIIDX (Double)                                                          |
 | 18K                                                    | E5 - Dancing Super Station (Double).                                            |
-| <span style="text-decoration:line-through;">20K</span> | <span style="text-decoration:line-through;"></span>                             |
+| 20K                                                    |                                                                                 |
 
 Not included: Keyboardmania (Light+, Real) \[24K\] and (Double) \[48K\].
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!mania/en.md]: add empty line after this

File: wiki/Ranking_Criteria/osu!taiko/en.md [hunk 17]
```diff
@@ -1,19 +1,20 @@
+osu!taiko Ranking Criteria
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/en.md]: change to `osu!taiko` (explanation at osu!)

File: wiki/Ranking_Criteria/osu!taiko/en.md [hunk 18]
```diff
@@ -1,19 +1,20 @@
...
+===========================
+
 These rules and guidelines has been discussed in the [Taiko Discussion Thread](http://osu.ppy.sh/forum/p/1275323). New rules or guidelines may be suggested in the [discussion forum](http://osu.ppy.sh/forum/87) which are going to be updated once the discussion reaches an agreement.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/en.md]: use https

File: wiki/Ranking_Criteria/osu!taiko/en.md [hunk 19]
```diff
@@ -22,42 +23,24 @@ Gameplay
...
-
 -   **Use drum-rolls (slider) sparingly within a map and keep a 1/2 beat space between them and any other object.** Avoid using sliders for 125 BPM songs or lower (as the ticks change to 1/8) unless the music calls for it. Use a tick rate of 3 if the song self is snapped on 1/3.
-
-<!-- -->
-
 -   **Try to leave at least a 1/2 beat between a note and a spinner.** Using them on streams may be okay. Short spinners are highly not recommended, since they cover the notes and can cause players to miss due to pressing too much.
-
-<!-- -->
-
 -   **Custom hitsounds must be drum-related.** If custom hitsounds are used, they should have deeper/lower tone set as don and a higher/lighter tone as kat.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/en.md]: add empty line to after this line

File: wiki/Ranking_Criteria/en.md [hunk 20]
```diff
@@ -1,23 +1,24 @@
...
+------
 
 All rules are exactly that: RULES. They are NOT guidelines and may NOT be broken under ANY circumstance.
 
-General
--------
+### General
 
 -   **No two hitobjects on the same tick (osu!mania game mode is an exception).** This includes hit circles, starts and ends of sliders, and starts and ends of spinners. Although this may be possible to accomplish while playing, it defeats the purpose of following the rhythm of the song.
 -   **Do not overuse kiai time.** The general rule of thumb is one toggle per minute of playtime. Kiai is meant to accent chorus sections, so don't use it just for fountains unless you're doing this rarely. You can toggle kiai off and back on halfway through the chorus if it fits, but using it constantly every few beats/measures is just ridiculous and can distract players.
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/en.md]: Idk, i wont change the content, because I mustn't

Comment (MegaApplePi) [inline wiki/Ranking_Criteria/en.md]: > just saying...

I was just saying, no action needed :s

**Pull request merged**